addons/web/static/src/core/utils/functions.js

```diff
@@ -31,3 +31,15 @@ export function uniqueId(prefix = "") {
 }
 // set nextId on the function itself to be able to patch then
 uniqueId.nextId = 0;
+
+/**
+ * Generate a unique identifier (64 bits) in hexadecimal.
+ *
+ * @returns {string}
+ */
+export function uuid() {
+    const array = new Uint8Array(8);
+    window.crypto.getRandomValues(array);
+    // Uint8Array to hex
+    return [...array].map((b) => b.toString(16).padStart(2, "0")).join("");
+}
```

Comment:
I would put it in strings.js though, not functions.js which was supposed to contain helpers related to functions (like memoize). Having uniqueId here is already a mistake.